web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -197,6 +197,10 @@ function getNavSubsectionForRoute(
       })
     );
 
+  if (!!feature.parent) {
```

Comment:
```suggestion
  if (feature.parent) {
```

Can we also leave a comment on what it does?